autoapi/mappers/python/objects.py

```diff
@@ -100,6 +100,22 @@ def display(self):
 
         :type: bool
         """
+        try:
+            skip = (
+                self.app.emit_firstresult(
+                    "autoapi-skip-member",
+                    self._class_content,
+                    self.name,
```

Comment:
`self.full_name` stores the fully qualified name rather than `self.name`.

Reply:
There is no such attribute `self.full_name` on `PythonMapperBase` instance, maybe you mean `self.id` attribute?

Reply:
Woops. Yes I do mean `self.id`!